tests/unit/environments/test_models.py

```diff
@@ -0,0 +1,58 @@
+from datetime import datetime, timedelta
+
+from flag_engine.environments.models import EnvironmentAPIKeyModel
+
+
+def test_environment_api_key_model_is_valid_is_true_for_non_expired_active_key():
+    assert (
+        EnvironmentAPIKeyModel(
+            id=1,
+            key="scr.random_key",
```

Comment:
```suggestion
            key="ser.random_key",
```

Reply:
Probably not important but it would be good to be consistent with this. It's also evident in the 3 tests below.